packages/components/organisms/f-header/src/tenants/en-AU.js

```diff
@@ -49,5 +49,8 @@ export default {
     offers: {
         text: 'For you',
         url: '/offers'
-    }
+    },
+    selectYourCountryText: 'Select your country',
+    currentCountryKey: 'au',
+    changeCurrentCountry: 'You are on the AU website, click here to change'
```

Comment:
Is AU approved copy here?
```suggestion
    changeCurrentCountry: 'You are on the Australian website, click here to change'
```

Reply:
I took it from a provided translations file